tests/integration/common/checks-after-conversion/test_enabled_repositories.py

```diff
@@ -21,22 +21,38 @@ def _check_enabled_repos_rhel8(enabled_repos: str = "", eus: bool = False):
     assert appstream_repo in enabled_repos
 
 
+def _check_enabled_repos_rhel7(enabled_repos: str = "", els: bool = False):
+    """Helper function to assert correct RHEL7 repositories are enabled after the conversion."""
+    repo = ""
+    if els:
+        repo = "rhel-7-server-els-rpms"
+    else:
+        repo = "rhel-7-server-rpms"
+
+    assert repo in enabled_repos
+
+
 @pytest.mark.test_enabled_repositories
 def test_enabled_repositories(shell):
     """
-    Verify that the correct repositories (including EUS if applies) are enabled after the conversion.
+    Verify that the correct repositories (including EUS/ELS if applies) are enabled after the conversion.
     """
 
     try:
         enabled_repos = shell("yum repolist").output
         system_release = SystemInformationRelease()
         assert "redhat" in system_release.distribution
 
-        if system_release.version.major == 7 and system_release.version.minor == 9:
-            assert "rhel-7-server-rpms/7Server/x86_64" in enabled_repos
+        if system_release.version.major == 7:
```

Comment:
Pasting the suggestion for this whole if-else section, but what do you think about: 

```suggestion
        if system_release.version.major == 7:
            # Handle the special test case scenario where we use the
            # account with the ELS repositories available
            is_els = os.path.exists("/els_repos_used")
            _check_enabled_repos_rhel7(enabled_repos, els=is_els)
        elif system_release.version.major == 8:
            # Handle the special test case scenario where we use the
            # account with the EUS repositories available
            is_eus = os.path.exists("/eus_repos_used")
            _check_enabled_repos_rhel7(enabled_repos, els=is_eus)
```

Reply:
Smart! I'll do that, thank you, @r0x0d